lib/viewmodels/home_viewmodel.dart

```diff
@@ -93,15 +93,17 @@ class HomeViewModel extends BaseModel {
 
   Future logoutAndClearData() async {
     _dialogService.showCustomProgressDialog(title: 'Logging You Out');
-    await _userApi.userLogout();
+    await _userApi.userLogout().catchError((err) {
+      print('error in logout: $err');
```

Comment:
No need to catch error here.

Reply:
We will add a better error handling later

Reply:
Without catching the exception app gets stuck at `Logging You out`.

Reply:
Yeah, so let's use try catch block here.